app/jobs/reports/identity_verification_report.rb

```diff
@@ -13,6 +13,40 @@ def perform(report_date)
       csv = report_maker.to_csv
 
       save_report(REPORT_NAME, csv, extension: 'csv')
+
+      if emails.empty?
+        Rails.logger.warn 'No email addresses received - Identity Verification Report NOT SENT'
+        return false
+      end
+
+      emails.each do |email|
+        ReportMailer.tables_report(
+          email: email,
+          subject: "Daily Identity Verification Report - #{report_date.to_date}",
+          reports: reports,
+          message: preamble,
+          attachment_format: :csv,
+        ).deliver_now
+      end
+    end
+
+    def preamble
+      <<~HTML.html_safe # rubocop:disable Rails/OutputSafety
+        <h2>
+          Identity Verification Report
+        </h2>
+        <p>
+          Disclaimer: This Report is In Progress: Not Production Ready
+        </p>
+      HTML
+    end
+
+    def emails
+      [IdentityConfig.store.team_ada_email]
```

Comment:
This works fine for now, and I'm wondering if in the long run it would work better to expect a single email address that's a mailing list, and add anyone who's interested to that list, rather than requiring a code change & deploy to add/remove people.

Reply:
We could probably create another email alias in the future like `identity_reports@...gov`. We still have more things to do in this epic.